deploy/manifests/node-plugin-daemonset.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
       - name: driver-registrar
-        image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.2.0
+        image: harbor.cloud.netease.com/curve/csi-node-driver-registrar:v2.2.0
```

Comment:
necessary to change registry.k8s.io/sig-storage images?

Reply:
done